### DISCUSSION TITLE
Stash for app assigned for All Desktops

### DISCUSSION THREAD

**Discussion by rinodrops** [rxhanson/RectanglePro-Community#627] [Ideas]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: macOS 14.0
Rectangle version: Pro v3.0.2 (143)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

The idea of Stash is letting an app always sits on the desktop to access it easily
but does not affect the currently used app.
I want to use ForkLift while using the other apps, so I assign ForkLift to All Desktops
using `Dock | Options` and stash it. However, the window is stashed only in one space,
even if the app itself is assigned to All Desktops.

**Comment by rxhanson** [comment]
Thanks for reporting. There are some macOS limitations getting in the way here, and at the moment I don't have a good workaround. If I can figure something out, I'll post back here.

**Comment by alexanderblackh** [comment]
Any update on this? This is the only thing holding me back from using it personally. Currently using Slidepad but it's less than ideal, rather have my native windows. Maybe doing something similar to bartender and taking advantage of screen recordings??

**Reply by rxhanson** [reply]
Thanks for asking. Unfortunately, I still haven't found away around what macOS is doing here :(

I don't think screen recording capabilities would give us what we're looking for, but I'll give this some more thought. Maybe there's something else I haven't thought of yet.

**Reply by alexanderblackh** [reply]
It's less then ideal, but what about only supporting this feature for apps where the users has enabled "All Desktops"?
<img width="313" alt="image" src="https://github.com/user-attachments/assets/ff1edccb-43f4-4fb8-b81d-c492b07171c3" />

I haven't worked with MacOS's windowing API's as I'm a React Native developer, but I'll take a look. Is this the right documentation? https://developer.apple.com/documentation/coregraphics/quartz_window_services

**Reply by alexanderblackh** [reply]
Maybe it could be an advanced feature and use a virtual desktop to capture said windows?
